static/js/containers/DashboardPage.js

```diff
@@ -686,6 +702,11 @@ class DashboardPage extends React.Component {
       />;
     }
 
+    let learnersInProgramCard;
+    if (R.pathOr(false, [program.id, 'data', 'learners'],programLearners)) {
+      learnersInProgramCard = <LearnersInProgramCard programLearners={programLearners[program.id].data}/>;
+    }
```

Comment:
not sure if I did what you wanted https://github.com/mitodl/micromasters/pull/3335/commits/c33019da2bb10078b93ab2e856b53ec5fac6df0e